src/components/AddPlayerButton.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import { ContentCardAction } from "./ContentCard";
+
+class AddPlayerButton extends Component {
+    constructor(props) {
+        super(props);
+
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleSubmit(event) {
+        this.props.addingPlayer(true);
+        event.preventDefault();
+    }
+
+    render() {
+        return (
+            <form onSubmit={this.handleSubmit}>
```

Comment:
Having a form is probably overkill and will give you functionality for 'submit on hitting enter' which seems a bit weird here. With the whole button and input switching in its own component, have that component pass a function down and keep this one only UI and use a functional component

```js
const AddPlayerButton = ({ onClick }) => <button onClick={onClick}>Add Player</button>
```
This could also just be made into a Button component
```js
const Button = ({ buttonText, onClick }) => <button onClick={onClick}>{buttonText}</button>
```
That we could use elsewhere.